lib/conda-recipe/meta.yaml

```diff
@@ -0,0 +1,80 @@
+# Copyright 2018-2022 Streamlit Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# NOTE: Our conda release of streamlit currently requires Python 3.8+ while
+# the PyPI release requires Python 3.7+, so we can't grab the minimum python
+# version from package_data.get('python_requires') for now.
+{% set python_version_bound = ">=3.8" %}
+{% set package_data = load_setup_py_data() %}
+
+
+package:
+  name: {{ package_data.get('name')|lower }}
+  version: {{ package_data.get('version') }}
+
+source:
+  path: ..
+
+build:
+  # TODO(vdonato): Figure out a good way to increment this build number
+  # automatically? Not sure if this is strictly necessary.
+  number: 0
```

Comment:
The GIT_HASH being piped in here makes the build number unnecessary, I think? (GIT_HASH will uniquify the package name already. I don't think the number adds anything else we care about.)

Reply:
Ack, I'll add a comment mentioning that it's set because it's required but isn't really used for anything